at_secondary/at_secondary_server/lib/src/verb/handler/scan_verb_handler.dart

```diff
@@ -173,11 +175,20 @@ class ScanVerbHandler extends AbstractVerbHandler {
   /// Checks if a key starts with `public:_`, `private:`, `privatekey:`.
   /// [key] : The key to check.
   /// Returns true if key starts with pattern, else false.
-  bool _isPrivateKeyForAtSign(
-    String key,
-  ) {
+  ///
+  /// When showHidden is set true, display hidden keys.
+  bool _isPrivateKeyForAtSign(String key, bool showHiddenKeys) {
+    // If showHidden is set to true, display hidden public keys/self hidden keys.
+    // So returning false
+    // public hidden key: public:__location@alice
+    // self hidden key: _location@alice
+    if ((key.toString().startsWith('public:__') || key.startsWith('_')) &&
+        showHiddenKeys) {
```

Comment:
Please put round braces around the && condition for readability

Reply:
Sure Gary.